test/config/PB2NCConfig_pbl

```diff
@@ -0,0 +1,163 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// PB2NC configuration file.
+//
+// For additional information, see the MET_BASE/config/README file.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+//
+// PrepBufr message type
+//
+message_type = ["ONLYSF", "ADPUPA"];
+
+//
+// Mapping of message type group name to comma-separated list of values
+// Derive PRMSL only for SURFACE message types
+//
+message_type_group_map = [
+   { key = "SURFACE"; val = "ADPSFC,SFCSHP,MSONET";               },
+   { key = "ANYAIR";  val = "AIRCAR,AIRCFT";                      },
+   { key = "ANYSFC";  val = "ADPSFC,SFCSHP,ADPUPA,PROFLR,MSONET"; },
+   { key = "ONLYSF";  val = "ADPSFC,SFCSHP";                      }
+];
+
+//
+// Mapping of input PrepBufr message types to output message types
+//
+message_type_map = [];
+
+//
+// PrepBufr station ID
+//
+station_id = [];
+
+////////////////////////////////////////////////////////////////////////////////
+
+//
+// Observation time window
+//
+obs_window = {
+   beg = -2700;
+   end =  2700;
+}
+
+////////////////////////////////////////////////////////////////////////////////
+
+//
+// Observation retention regions
+//
+mask = {
+   grid = "";
+   poly = "";
+}
+
+////////////////////////////////////////////////////////////////////////////////
+
+//
+// Observing location elevation
+//
+elevation_range = {
+   beg =  -1000;
+   end = 100000;
+}
+
+////////////////////////////////////////////////////////////////////////////////
+
+//
+// Observation types
+//
+pb_report_type  = [ 120, 220, 221, 122, 222, 223, 224, 131, 133, 233, 153, 156, 157, 180, 280, 181, 182, 281, 282, 183, 284, 187, 287 ];
+
+in_report_type  = [];
+
+instrument_type = [];
+
+////////////////////////////////////////////////////////////////////////////////
+
+//
+// Vertical levels to retain
+//
+level_range = {
+   beg = 1;
+   end = 511;
+}
+
+level_category = [0, 1, 4, 5, 6];
+
+///////////////////////////////////////////////////////////////////////////////
+
+//
+// BUFR variable names to retain or derive.
+// Use obs_bufr_map to rename variables in the output.
+// If empty, process all available variables.
+//
+obs_bufr_var = ["TOB", "UOB", "VOB", "TOCC", "D_RH", "TDO", "PMO", "HOVI", "CEILING", "MXGS", "D_CAPE", "D_PBL"];
+//obs_bufr_var = ["TOB", "UOB", "VOB", "TOCC", "D_RH", "TDO", "PMO", "HOVI", "CEILING", "MXGS", "D_CAPE"];
```

Comment:
```suggestion
```
Rather than commenting out a previous setting, I'd prefer to remove it entirely to keep the config file as clean as possible.